src/parser/flow_lexer.ml

```diff
@@ -58,6 +58,14 @@ let scinumber = [%sedlex.regexp?
 let wholenumber = [%sedlex.regexp? underscored_digit, Opt '.']
 let floatnumber = [%sedlex.regexp? Opt underscored_digit, '.', underscored_decimal]
 
+let binbigint = [%sedlex.regexp? '0', ('B' | 'b'), underscored_bin, 'n']
```

Comment:
I'm not too well versed in sedlex, but can you write these regexes in terms of the above regexes? Like

```
let binbigint = [%sedlex.regexp? binnumber, 'n']
let octbigint = [%sedlex.regexp? octnumber, 'n']
...
```

Reply:
Yes, it works